package common.protocol.L2G;

message PlayerLogout
{
	required int64 roleid = 1;
	required int32 linkid = 2;
}
